addons/l10n_id_efaktur_coretax/models/product_template.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    l10n_id_product_code = fields.Many2one("l10n_id_efaktur_coretax.product.code",
+        default=lambda self: self.env.ref('l10n_id_efaktur_coretax.product_code_000000_goods', raise_if_not_found=False),
+        string="E-Faktur Product Code")
```

Comment:
```suggestion
    l10n_id_product_code = fields.Many2one(
        comodel_name="l10n_id_efaktur_coretax.product.code",
        default=lambda self: self.env.ref('l10n_id_efaktur_coretax.product_code_000000_goods', raise_if_not_found=False),
        string="E-Faktur Product Code"
    )
```
If you are going to use multiple lines for the field, then align everything, it's easier to read